google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITTracingTest.java

```diff
@@ -247,6 +252,29 @@ void assertHasExpectedAttributes(SpanData spanData, String... additionalExpected
     }
   }
 
+  // Returns true if an only if the given span data contains an event with the given name and the
+  // given expected
+  // attributes.
+  boolean hasEvent(SpanData spanData, String eventName, @Nullable Attributes expectedAttributes) {
+    if (spanData == null) {
+      return false;
+    }
+
+    List<EventData> events = spanData.getEvents();
+    for (EventData event : events) {
+      if (event.getName().equals(eventName)) {
+        if (expectedAttributes == null || expectedAttributes.isEmpty()) {
```

Comment:
is this necessary? May be easier to read to just do the equals check below.

Reply:
sorry, guess you still nee the null check but besides that I would leave it to the equals

Reply:
Done